docs/sdk/access-control/lit-action-conditions.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 4
 
 You can create a Lit Action Condition to grant access whenever a given Lit Action returns true. Lit Actions are JS code that can be executed on the Lit Protocol network. You can use Lit Actions to create custom access control conditions.
 
-## Lit Action must return true
+## An Example of Lit Action Conditions
```

Comment:
For the example included here, it seems the previous title was appropriate. 

I understand we don't want to give the impression that 'must return true' is the only type of condition that can be configured using Lit Actions, so maybe we want to add more context in the caption above? (as it still says 'you can create a Lit Action Condition to grant access whenever a given Lit Action returns true')

maybe for this title, we use 'Example: Lit Action must return 'true' ' and then add more context above?